src/test/denialofservice_tests.cpp

```diff
@@ -146,7 +146,14 @@ BOOST_AUTO_TEST_CASE(stale_tip_peer_management)
         BOOST_CHECK(node->fDisconnect == false);
     }
 
-    SetMockTime(GetTime() + 3*consensusParams.nPowTargetSpacing + 1);
+    // STALE_CHECK_INTERVAL is used in PeerLogicValidation::CheckForStaleTipAndEvictPeers
+    // as minimal time to check tip stale i.e. 10 minutes
+    // we use maximum value of STALE_CHECK_INTERVAL and nPowTargetSpacing
+    // NOTE: STALE_CHECK_INTERVAL is static that why we use raw value 10 * 60, sync may need in future
+
+    auto time = std::max({static_cast<int64_t>(10 * 60), 3 * consensusParams.nPowTargetSpacing});
```

Comment:
braces?

Reply:
Could be wrong, but I think this was a test @bvbfan worked on.  In any case, std::max can take an initializer list, so there's really no difference (functionally) if they're removed other than style preference.